src/test/java/com/eleks/academy/whoami/core/impl/PersistentPlayerTest.java

```diff
@@ -0,0 +1,22 @@
+package com.eleks.academy.whoami.core.impl;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.Future;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class PersistentPlayerTest {
+
+	@Test
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.